Improvement of Redis setup (#2432)
owncloud/docs: modules/admin_manual/pages/configuration/server/caching_configuration.adoc

[source,php,subs="attributes+"]
----
'redis' => [

>     'host' => 'localhost',       // For a unix domain socket => '/var/run/redis/redis.sock'

Reviewer: ```suggestion
    'host' => 'localhost',       // For a unix domain socket 'host ' => '/var/run/redis/redis.sock'
```
Maybe this will make it clearer that the key stays as `host` and the value should be `/var/run/redis/redis.sock`

],
----

> For setting up Redis to be accessed via a socket from from webserver user, consider the following settings:

Reviewer: ```suggestion
If setting up Redis to be accessed via a socket from from webserver user, then consider the following settings:
```
(and are these just recommendations to consider, or do you actually have to do these?

. Make the webserver user `www-data` member of the group `redis` in `/etc/group`: `redis:x:110:www-data`
. Set in your Redis configuration `/etc/redis/redis.conf` the `unixsocketperm` to `770`

> For a comparison benachmark to see the differences run:

Reviewer: ```suggestion
For a comparison benchmark to see the differences run:
```
